style.css

```diff
@@ -0,0 +1,269 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: sans-serif;
+}
+
+body {
+  background-color: #c996cc;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
why giving this to body ?

Reply:
@faran4engg For just this single page project I given to the body

Reply:
no, body should not have these css @Naufil-Shemle

Reply:
OK Sir, updated.